digital_land/api.py

```diff
@@ -0,0 +1,92 @@
+import csv
+import os
+import requests
+import logging
+import typing
+
+DEFAULT_URL = "https://files.planning.data.gov.uk"
+
+
+class API:
+    def __init__(
+        self, specification, url: str = DEFAULT_URL, cache_dir: str = "var/cache"
```

Comment:
question: Do we always pass https://files.planning.data.gov.uk as the URL, what about when it is being executed on the dev/staging envt?